config.lua

```diff
@@ -13,25 +13,26 @@ Config.Language = {
     NotHoldingAnimal = "You Got Nothing To Sell",
     NotInTheButcher = "I won't buy that animal",
     AnimalSold = "Thanks for the deal you got ",
-    SkinnableAnimalstowed = "You obtained ",
+    SkinnableAnimalstowed = "Obtained ",
     stow = "Stow",
     press = "Press",
     sell = "Sell",
     notabutcher = "You Dont Have The Right Job",
     dollar = " $",
-    join = " and ",
+    join = ", ",
     FullInventory = "Inventory is too full."
 }
 
-
 ------------------- PROMPT -----------------
 Config.keys = {
     G = 0x760A9C6F, -- butcher sell and Stow
 }
 
+Config.aiButcherped = true -- spawn ai butched ped set to false if you dont want an ai butcher ped to spawn
+
 Config.joblocked = false -- lock the butcher so only people with the job can access. u can change access to each butcher by editing this  butcherjob = "butcher"
 
-Config.maxpelts = 2 -- max pelts allowed on back of horse (Max 3 allowed)
+Config.maxpelts = 5 -- max pelts allowed on back of horse
```

Comment:
Everything looks great, but can you please put this back to what its supposed to be. The limit is 3 for technical limitations.

```lua
Config.maxpelts = 2 -- max pelts allowed on back of horse (Max 3 allowed)
```

Reply:
Sure, Limit changed to 3

Reply:
added the butcherped values

Reply:
changed notification to new API